sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -657,7 +664,7 @@ private bool TryGetAttributeProperties(AttributeData attributeData, Location? at
                 return true;
             }
 
-            private bool TryLoadConstructorArguments(AttributeData attributeData, IDictionary<string, object?> dict, Location? attribLocation)
+            private bool TryLoadConstructorArguments(AttributeData attributeData, IDictionary<string, object?> argsDict, Location? attribLocation)
```

Comment:
nit: since you're renaming, let's just name this `arguments`

Reply:
also rename `attribLocation` -> `attributeLocation` or simply `location` (the former sounds better to me)